baseline-refaster-rules/src/main/java/com/palantir/baseline/refaster/MockitoVerifyNoInteractions.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.refaster;
+
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.verifyZeroInteractions;
+
+import com.google.errorprone.refaster.ImportPolicy;
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import com.google.errorprone.refaster.annotation.Repeated;
+import com.google.errorprone.refaster.annotation.UseImportPolicy;
+
+public final class MockitoVerifyNoInteractions {
+
+    @BeforeTemplate
+    @SuppressWarnings("deprecation")
+    // must use @Repeated for varargs per https://github.com/google/error-prone/issues/568
+    void before(@Repeated Object varargs) {
+        verifyZeroInteractions(varargs);
+    }
+
+    @AfterTemplate
+    @UseImportPolicy(ImportPolicy.STATIC_IMPORT_ALWAYS)
+    void after(@Repeated Object varargs) {
+        verifyNoInteractions(varargs);
```

Comment:
As per javadoc on this stuff, it should be replacing it with verifyNoMoreInteractions. Granted that those types of tests are probably a bit iffy, but this replacement probably means people's excavators will back up.

Reply:
Althought that might be an argument for this change though, to make people look at those tests again.

Reply:
@dansanduleac fixed this yesterday: https://github.com/palantir/gradle-baseline/pull/975/files